tests/inference/python_test_configs/generate_configs.py

```diff
@@ -65,7 +65,7 @@
 
 
 # Generate incremental decoding configs
-all_models = llama_models + opt_models + falcon_models + mpt_models
+all_models = llama_models + opt_models + mpt_models
```

Comment:
Any reason we disabled the falcon model tests?

Reply:
falcon 7B full precision memory inefficient, I'll let it run half precision before merging.